js/shared.js

```diff
@@ -1111,6 +1111,279 @@ export const allCountries = {
     "Zimbabwe":240
 }
 
+export const country3Codes = [
```

Comment:
Should we use the 3-letter code in `conceptIdMapping.countries` (or en.js, es.js), so that we don't need to maintain a separate list here?